src/360-core/browzine-360-core-adapter.js

```diff
@@ -214,43 +214,36 @@ browzine.serSol360Core = (function() {
     return template;
   };
 
-  function searchTitles(titles, callback) {
-    var title = titles.shift();
-
-    if (title) {
-      $.getJSON(title.endpoint, function (response) {
-        var data = getData(response);
-        var browzineWebLink = getBrowZineWebLink(data);
-        var coverImageUrl = getCoverImageUrl(data);
-        var defaultCoverImage = isDefaultCoverImage(coverImageUrl);
-
-        if (browzineWebLink && showJournalBrowZineWebLinkText()) {
-          var template = browzineWebLinkTemplate(browzineWebLink);
-          $(title.target).find(".results-identifier").append(template);
-        }
-
-        if (coverImageUrl && !defaultCoverImage && showJournalCoverImages()) {
-          var resultsTitleImageContainerSelector = ".results-title-image-div";
-          var resultsTitleImageSelector = ".results-title-image-div img.results-title-image";
-          var boxShadow = "1px 1px 2px #ccc";
-
-          if ($(title.target).find(resultsTitleImageSelector).length > 0) {
-            $(title.target).find(resultsTitleImageSelector).attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", boxShadow);
-          } else {
-            $(title.target).find(resultsTitleImageContainerSelector).append("<img alt='Results Title Image' class='results-title-image'/>");
-            $(title.target).find(resultsTitleImageSelector).attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", boxShadow);
+  function searchTitles(titles) {
+
+    titles.forEach(title => {
```

Comment:
It looks like IE has supported this since Sep 2012 - https://www.w3schools.com/jsref/jsref_forEach.asp - so I think we're in good shape.  
(MDN's [page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) doesn't even mention IE anymore, heh)

If someone does write in and says things break after this, though, we should be prepared to possibly change from `forEach` to something else, like a regular `for(int i = 0; i < title.length; i++) { ... }` approach.

So I'm going to add a deploy information note here that we should watch out for any reports of this entirely breaking being written in, and if so, change it.  

It shouldn't happen, but juuuust in case.
 
But hooray for getting rid of the recursion, no real need for it here I think 👍